src/Umbraco.Web.UI.Client/src/packages/user/user/collection/views/grid/user-grid-collection-view.element.ts

```diff
@@ -145,19 +145,28 @@ export class UmbUserGridCollectionViewElement extends UmbLitElement {
 
 			#user-grid {
 				display: grid;
-				grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
+				grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
 				gap: var(--uui-size-space-4);
 			}
 
 			uui-card-user {
 				width: 100%;
-				height: 180px;
 				justify-content: normal;
 				padding-top: var(--uui-size-space-5);
+				flex-direction: column;
 			}
 
 			.user-login-time {
-				margin-top: auto;
+				display: -webkit-box;
+				-webkit-line-clamp: 2;
+				-webkit-box-orient: vertical;
+				overflow: hidden;
+				text-overflow: ellipsis;
+				vertical-align: center;
+				margin-top: 3px;
+				font-weight: 700;
```

Comment:
We should not make the date bold, so please remove the font-weight.

Reply:
But please see my comment on the thread, I think we should get rid of the ellipsis. Just leaving the margin-top as the only defined property for this.